caikit/runtime/model_management/model_manager.py

```diff
@@ -192,16 +334,11 @@ def unload_model(self, model_id) -> int:
 
         return model_size
 
-    def unload_all_models(self) -> None:
```

Comment:
This is only ever used in unit tests, so you could argue that it should be removed entirely. As it was, it was missing logic from `unload_model` and was doing too much exception handling